stumpy/stomp.py

```diff
@@ -100,13 +100,35 @@ def _stomp(T_A, m, T_B=None, ignore_trivial=True):
     else:
         if ignore_trivial:
             P, I = stamp._mass_PI(
-                T_A[:m], T_B, M_T, Σ_T, T_subseq_isconstant, 0, excl_zone
+                T_A[:m],
+                T_B,
+                M_T,
+                Σ_T,
+                0,
+                excl_zone,
+                T_subseq_isconstant=T_subseq_isconstant,
+                Q_subseq_isconstant=Q_subseq_isconstant[[0]],
```

Comment:
I haven't added support for `subseq_isconstant` to `stomp.py` yet. According to its docstring, this function is not supported and is only here for reference. So, should we add `subseq_isconstant` param to it? If not, then we can just avoid using `T_subseq_isconstant` and `Q_subseq_isconstant` within this function and do not pass them to `stamp._mass_PI`.

Reply:
Yes, please skip stomp